woocommerce/woocommerce-gutenberg-products-block
# Persist previous shipping rates while loading and show its package name

This PR might be easier to review commit-by-commit:
* 18c05cf0d32f5701f792036b1f935b223d042521 fixes #1801.
* e20c23ee131f0e9bba6b5e932c8512c72a14f67b refactors `ShippingRatesControl` to use `LoadingMask`'s `isLoading` prop.
* 367135e7a3652f39918afa2779d8378191715d1f fixes #1802 and updates styling to be closer to designs.

I can split them if required, but since all of them are quite related, I thought it might be better to test all of them at the same time.

### Screenshots

![Peek 2020-02-25 22-35](https://user-images.githubusercontent.com/3616980/75289738-3b479980-581f-11ea-83d8-7ad3fbc5b281.gif)
_(GIF showing three cases: no rates, several rates and only one rate)_

### How to test the changes in this Pull Request:

1. Create two posts one with a _Cart_ block and another one with the _Checkout_ block.
2. In both of them try changing the address and verify shipping rates persist while loading.
3. Verify the package name is shown on top when there is more than one package. To do this:
	* Install [WooCommerce Advanced Shipping Packages](https://woocommerce.com/products/woocommerce-advanced-shipping-packages/) and set up some package rules ([see how](https://docs.woocommerce.com/document/woocommerce-advanced-shipping-packages/)).
	* Add products that will be in different packages to your cart.
